nomenclature/definitions/variable/technology/power-plant.yaml

```diff
@@ -230,4 +230,12 @@ Storage Type|Electricity|Energy Storage System:
 
 Stored Energy Inventory|Electricity|Energy Storage System:
    description: State of the stored energy by ESS
-   unit: [MWh, GWh]
\ No newline at end of file
+   unit: [MWh, GWh]
+   
+Energy Charge|Electricity|Energy Storage System:
```

Comment:
The "Charge" of a battery (the energy level) is different from "Charging a battery", so this variable name is not ideal. Any suggestions for alternatives? Also, the description should be more specific.